man/memkind.3

```diff
@@ -552,7 +552,7 @@ is available or an error code from the
 section if it is not.
 .PP
 .BR memkind_check_dax_path ()
-retruns zero if file-backed kind memory in the specified directory path
+returns zero if file-backed kind memory in the specified directory path
```

Comment:
Missing a verb here...
Returns zero if file-backed kind memory is (?) in the specified directory path

Reply:
Done in #507 ( after merge #507 it will be available in this PR)